e2e/test/helpers/AmqpConnectionStatusChange.cs

```diff
@@ -8,21 +8,19 @@ namespace Microsoft.Azure.Devices.E2ETests.Helpers
     public class AmqpConnectionStatusChange
     {
         private readonly string _deviceId;
-        private readonly MsTestLogger _logger;
 
-        public AmqpConnectionStatusChange(string deviceId, MsTestLogger logger)
+        public AmqpConnectionStatusChange(string deviceId)
         {
             ConnectionStatusChangeCount = 0;
             _deviceId = deviceId;
-            _logger = logger;
         }
 
         public int ConnectionStatusChangeCount { get; set; }
 
         public void ConnectionStatusChangeHandler(ConnectionStatusInfo connectionStatusInfo)
         {
             ConnectionStatusChangeCount++;
-            _logger.Trace($"{nameof(AmqpConnectionStatusChange)}.{nameof(ConnectionStatusChangeHandler)}: {_deviceId}: status={connectionStatusInfo.Status} statusChangeReason={connectionStatusInfo.ChangeReason} count={ConnectionStatusChangeCount}");
+            VerboseTestLogger.WriteLine($"{nameof(AmqpConnectionStatusChange)}.{nameof(ConnectionStatusChangeHandler)}: {_deviceId}: status={connectionStatusInfo.Status} statusChangeReason={connectionStatusInfo.ChangeReason} count={ConnectionStatusChangeCount}");
```

Comment:
I figured I'd take the only logger remaining and replace existing traces with it.

Thoughts?